cmd/istio-install/main.go

```diff
@@ -4,38 +4,26 @@
 package main
 
 import (
+	istioclient "github.com/kyma-project/istio/operator/internal/reconciliations/istio"
 	"os"
 	"time"
 
 	"istio.io/istio/istioctl/pkg/install/k8sversion"
 	istio "istio.io/istio/operator/cmd/mesh"
-	"istio.io/istio/operator/pkg/util/clog"
 	"istio.io/istio/pkg/kube"
-	istiolog "istio.io/istio/pkg/log"
 	"k8s.io/client-go/rest"
 )
 
-func initializeLog() *istiolog.Options {
-	logoptions := istiolog.DefaultOptions()
-	logoptions.SetOutputLevel("validation", istiolog.ErrorLevel)
-	logoptions.SetOutputLevel("processing", istiolog.ErrorLevel)
-	logoptions.SetOutputLevel("analysis", istiolog.WarnLevel)
-	logoptions.SetOutputLevel("installation", istiolog.WarnLevel)
-	logoptions.SetOutputLevel("translator", istiolog.WarnLevel)
-	logoptions.SetOutputLevel("adsc", istiolog.WarnLevel)
-	logoptions.SetOutputLevel("default", istiolog.WarnLevel)
-	logoptions.SetOutputLevel("klog", istiolog.WarnLevel)
-	logoptions.SetOutputLevel("kube", istiolog.ErrorLevel)
-
-	return logoptions
-}
-
 func main() {
 	iopFileNames := []string{os.Args[1]}
 
-	istioLogOptions := initializeLog()
-	registeredScope := istiolog.RegisterScope("installation", "installation")
-	consoleLogger := clog.NewConsoleLogger(os.Stdout, os.Stderr, registeredScope)
+	consoleLogger := istioclient.CreateIstioLibraryLogger()
+
+	if err := istioclient.ConfigureIstioLogScopes(); err != nil {
+		consoleLogger.LogAndError("Failed to configure Istio log: ", err)
+		os.Exit(1)
```

Comment:
That's a good question. For now I implemented it like that, because I considered running install without any output from install a bad condition. We simply don't know what happens. Also the likelihood that this will fail is very low. I think we can always adapt it later if we see that this is an issue.

Reply:
I agree, taking into account failing installation will abandon all the next steps (any harm impossible), this is a better alternative here.